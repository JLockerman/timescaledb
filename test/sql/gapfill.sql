-- Copyright (c) 2016-2018  Timescale, Inc. All Rights Reserved.
--
-- This file is licensed under the Apache License,
-- see LICENSE-APACHE at the top level directory.

\set ON_ERROR_STOP 0
SELECT locf(1);
SELECT interpolate(1);
SELECT time_bucket_gapfill(1,1,1,1);
\set ON_ERROR_STOP 1
