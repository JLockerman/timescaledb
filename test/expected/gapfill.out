-- Copyright (c) 2016-2018  Timescale, Inc. All Rights Reserved.
--
-- This file is licensed under the Apache License,
-- see LICENSE-APACHE at the top level directory.
\set ON_ERROR_STOP 0
SELECT locf(1);
ERROR:  function "locf" is not supported under the current license "ApacheOnly"
SELECT interpolate(1);
ERROR:  function "interpolate" is not supported under the current license "ApacheOnly"
SELECT time_bucket_gapfill(1,1,1,1);
ERROR:  function "time_bucket_gapfill" is not supported under the current license "ApacheOnly"
\set ON_ERROR_STOP 1
